### ISSUE TITLE
The level 1 optimization flag, tidySelectors, removes spaces inside the :not() pseudo-class

### ISSUE THREAD

**Issue by markadrake** [clean-css/clean-css#1221]
### Input CSS

```
.navigation__toggle:not(.navigation--mobile .navigation__toggle) {
    display: none;
}
```

### Actual output CSS

```
.navigation__toggle:not(.navigation--mobile.navigation__toggle){display:none}
```

### Expected output CSS

```
.navigation__toggle:not(.navigation--mobile .navigation__toggle){display:none}
```

### The Issue

With `tidySelectors` set to **true**, a critical `space` has been removed from my CSS combination selector:

`.navigation--mobile [space] .navigation__toggle`

**Comment by calvinjuarez** [comment]
Having this issue as well.

**Comment by jakubpawlowicz** [comment]
This issue is fixed in clean-css 5.1+. Hopefully you can upgrade.
